documentation/shared/attributes.adoc

```diff
@@ -21,7 +21,7 @@
 
 // Kafka upgrade attributes used in kafka upgrades section
 :DefaultKafkaVersion: 2.4.0
-:KafkaVersionLower: 2.3.1
+:KafkaVersionLower: 2.3.0
```

Comment:
`2.3.1` was the correct version here.

Reply:
Yes. I assumed a change was required here, but I've reverted back now.